src/core/CveDate.test.ts

```diff
@@ -56,11 +68,11 @@ describe(`CveDate`, () => {
     expect(differenceInCalendarDays(midnight,midnightYesterday)).toBe(1)
   });
 
-  it(`getYesterday() should output yesterday's date as a string`, async () => {
-    const now = new Date();
-    const yesterday = CveDate.getYesterday();
-    const yesterdate = parse(yesterday, 'yyyy-MM-dd', new Date());
-    expect(differenceInCalendarDays(now, yesterdate)).toBe(1);
-  });
+  // it(`getYesterday() should output yesterday's date as a string`, async () => {
```

Comment:
update to work based on iso timezone. ensure all date usage is based in gmt, and have any timezone conversions be on the user to manage.

Reply:
Default Yesterday to ISO